test/Chirp.Razor.UnitTest/ChirpStorageTests.cs

```diff
@@ -0,0 +1,141 @@
+            .Options;
+
+        var context = new ChirpDBContext(_contextOptions);
+        context.Database.EnsureCreated();
+
+        context.AddRange(
+            new Author { AuthorId = 1, Name = "Jens", Email = "test@mail.dk" },
+            new Author { AuthorId = 2, Name = "Børge", Email = "wow@dd.dk" }
+        );
+        context.SaveChanges();
+
+    }
+
+    [Fact]
+    public void QueryCheepCountReturnsCorrectCount()
+    {
+        var context = new ChirpDBContext(_contextOptions);
+        var chirpStorage = new ChirpStorage(context);
+
+        // Arrange
+        context.AddRange(
+            new Cheep { AuthorId = 1, Text = "Wow it works!", TimeStamp = DateTime.Now }
+        );
+        context.SaveChanges();
+
+        // Act
+        var count = chirpStorage.QueryCheepCount();
+
+        // Assert
+        Assert.Equal(1, count);
+    }
+
+    [Fact]
+    public void StoreCheepSavesCheep()
+    {
+        var context = new ChirpDBContext(_contextOptions);
+        var chirpStorage = new ChirpStorage(context);
+
+        // Arrange
+        var cheep = new Cheep
+        {
+            AuthorId = 1,
+            Text = "Another test cheep!",
+            TimeStamp = DateTime.Now
+        };
+
+        // Act
+        chirpStorage.StoreCheep(cheep);
+
+        // Assert
+        var storedCheep = context.Cheeps.FirstOrDefault();
+        Assert.NotNull(storedCheep);
+        Assert.Equal(cheep.Text, storedCheep.Text);
+    }
+
+    [Fact]
+    public void StoreCheepsSavesCheeps()
+    {
+        var context = new ChirpDBContext(_contextOptions);
+        var chirpStorage = new ChirpStorage(context);
+
+        // Arrange
+        List<Cheep> cheeps = new List<Cheep>{
+            new Cheep {
+                AuthorId = 1,
+                Text = "How u doin'?",
+                TimeStamp = DateTime.Now
+            },
+            new Cheep {
+                AuthorId = 2,
+                Text = "Hey all",
+                TimeStamp = DateTime.Now
+            }
+        };
+
+        // Act
+        chirpStorage.StoreCheeps(cheeps);
+
+        // Assert
+        var storedCheeps = context.Cheeps.ToList();
+        for (int i = 0; i < cheeps.Count; i++)
+        {
+            Assert.Equal(cheeps[i].Text, storedCheeps[i].Text);
+        }
+    }
+
+    [Fact]
+    public void QueryCheepsReturnsCheepsByAuthor()
+    {
+        var context = new ChirpDBContext(_contextOptions);
+        var chirpStorage = new ChirpStorage(context);
+
+        // Arrange
+        context.AddRange(
+            new Cheep { AuthorId = 1, Text = "Cheep 1", TimeStamp = DateTime.Now },
+            new Cheep { AuthorId = 1, Text = "Cheep 2", TimeStamp = DateTime.Now },
+            new Cheep { AuthorId = 2, Text = "Cheep 3", TimeStamp = DateTime.Now }
+        );
+        context.SaveChanges();
+
+        // Act
+        var cheeps = chirpStorage.QueryCheeps(0, 32, author: "Jens");
+
+        // Assert
+        Assert.Equal(2, cheeps.Count());
```

Comment:
Here we should consider checking that the actual values are actually what we're expecting them to be.
```suggestion
        // Arrange
        var cheepsToStore = new List<Cheep>
        {
            new() { AuthorId = 1, Text = "Cheep 1", TimeStamp = DateTime.Now },
            new() { AuthorId = 1, Text = "Cheep 2", TimeStamp = DateTime.Now },
            new() { AuthorId = 2, Text = "Cheep 3", TimeStamp = DateTime.Now }
        };
        context.AddRange(
            cheepsToStore
        );
        context.SaveChanges();

        // Act
        var cheeps = chirpStorage.QueryCheeps(0, 32, author: "Jens");

        // Assert
        cheeps.Should().BeEquivalentTo(cheepsToStore.Where(cheep => cheep.AuthorId == 1));
```